Microsoft.Toolkit.Uwp.SampleApp/Shell.xaml.cs

```diff
@@ -514,10 +520,40 @@ private void HamburgerMenu_OnOptionsItemClick(object sender, ItemClickEventArgs
             await Launcher.LaunchUriAsync(new Uri(e.Link));
         }
 
-        private void DocumentationTextblock_ImageResolving(object sender, ImageResolvingEventArgs e)
+        private async void DocumentationTextblock_ImageResolving(object sender, ImageResolvingEventArgs e)
         {
-            e.Image = new BitmapImage(new Uri("ms-appx:///Assets/pixel.png"));
-            e.Handled = true;
+            var deferral = e.GetDeferral();
+            BitmapImage image = null;
+
+            var absolute = Uri.TryCreate(e.Url, UriKind.Absolute, out Uri Url);
```

Comment:
If `absolute` is not going to be used again, just put the TryCreate in the if
`if(Uri.TryCreate(e.Url, UriKind.Absolute, out Uri url) == false)`